Add @node in typeDefs

MacondoExpress opened the pull request:
Add `@node` in the documentation for the types that are mapped to a Neo4j node as planned for future release.

MacondoExpress committed:
use @node in typeDefs

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-170.surge.sh

Updated pages:
- [Autogeneration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/autogeneration)
- [Custom directives](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-directives)
- [Custom logic](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-logic)
- [Database mapping](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/database-mapping)
- [Indexes and constraints](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/indexes-and-constraints)
- [Field configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/field-configuration)
- [Global configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/global-configuration)
- [Type configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/type-configuration)
- [Driver configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/driver-configuration)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

rsill-neo4j commented on modules/ROOT/pages/subscriptions/events.adoc:
```diff
@@ -747,7 +747,7 @@ subscription {
            }
         }
-    }
+    }As an example, consider these type definitions:
```
this is inside a code listing - intended?

MacondoExpress replied:
Nice catch!

rsill-neo4j commented on modules/ROOT/pages/directives/database-mapping.adoc:
```diff
@@ -183,7 +183,21 @@ The following query yields a different Cypher query depending on the user JWT:
     name
   }
-}ref::/directives/database-mapping.adoc#_declarerelationship[
+}
```
odd, this should be fixed in 5.x now

MacondoExpress replied:
The branch was not updated, after the update this is resolved!

MacondoExpress committed:
Merge remote-tracking branch 'upstream/5.x' into use-node-directive

MacondoExpress committed:
fix typo

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-170.surge.sh

Updated pages:
- [Autogeneration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/autogeneration)
- [Custom directives](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-directives)
- [Custom logic](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-logic)
- [Database mapping](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/database-mapping)
- [Indexes and constraints](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/indexes-and-constraints)
- [Field configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/field-configuration)
- [Global configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/global-configuration)
- [Type configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/type-configuration)
- [Driver configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/driver-configuration)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

MacondoExpress committed:
Explain importance of the @node directive

neo-technology-commit-status-publisher commented:
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.
<!-- Sticky Pull Request Commentdocs-pr-changes -->

MacondoExpress commented:
@rsill-neo4j  I've added a last commit with some rephrasing over the `@node` directive. I also moved the `@node` description to the top of the page as it will become the most fundamental block of the future version of the library. Can you have an extra check before merging it?

MacondoExpress commented:
@rsill-neo4j This is closed as we plan to add this to version 6.
